src/main/utils/translators.ts

```diff
@@ -3,11 +3,14 @@ import { GeneralInfo } from '../types/generalInfo';
 import { Issue } from '../types/issue';
 import { License } from '../types/license';
 import { Severity } from '../types/severity';
+import { GavGeneralInfo } from '../types/gavGeneralinfo';
 
 export class Translators {
-    public static toGeneralInfo(clientGeneral: IGeneral): GeneralInfo {
+    public static toGeneralInfo(clientGeneral: IGeneral): GeneralInfo | GavGeneralInfo {
```

Comment:
I don't think you have to specifically mention sub classes. Please remove `| GavGeneralInfo` also from all other places.
```suggestion
    public static toGeneralInfo(clientGeneral: IGeneral): GeneralInfo {
```